### PR TITLE
Portfolio: Mobile menu

### PR DESCRIPTION
**Pull request by lily4178993**
# Mobile Menu:
## Hamberger Menu

In this branch, I added an event listener that fires when a user resizes the window and displays the hamberger menu when the screen size is less than 768px.

You can see the live demo [here](https://lily4178993.github.io/Portfolio/)

### CODE REVIEW

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @lily4178993 , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your PR looks professional. ✔


## Required Changes ♻️

- [ ] Excellent work on the mobile menu :clap: . However, the design of your mobile menu differs from the template on Figma. Please correct this so that your design looks more like the template. :ok_hand: 😊

#### Your design
Screenshot from 2023-05-16 23-26-52


#### The Figma design
Screenshot from 2023-05-16 23-09-32


To assist you I would suggest you the below updates :point_down: 
```
menu-item {
    width: 90%;
    font-size: 1.5rem;
    font-weight: 700;
    line-height: 2rem;
    color: var(--color-lightsalmon);
    padding: 0 0 0.3em 0.5em;
    border-bottom: 1px solid #6F6C6B;
}
```
```
.menu-box:has(.menu:checked) .menu-list {
    width: 100%;
    display: flex;
    gap: 1em;
    flex-direction: column;
    align-items: center;
    margin-top: -20em;
}
```

 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily4178993** [comment]
Thank you @ridwanediallo for the review.

File: styles.css [hunk 1]
```diff
@@ -150,17 +150,21 @@
...
     font-weight: 700;
     line-height: 2rem;
     color: var(--color-lightsalmon);
     padding: 0 0 0.3em 0.5em;
-    border-bottom: 1px solid var(--color-whitesmoke);
+    border-bottom: 1px solid #f4f1f02b;
+  }
+
+  .menu-item:nth-last-child(1) {
+    display: none;
```
Comment (V-Blaze) [inline styles.css:167]: - [ ] Great job so far, :+1: :+1: You did great implementing some of the changes pointed out by the previous reviewer, But I think there are some improvements you need to make to your design in order to match the given design :wave: 
- - As pointed out by the previous reviewer, and I have also marked it on the screenshot below, your menu list items are meant to be at the top and not at the center, I'll suggest you update your styling to match the Figma menu design as part of the requirement fo this milestone is to replicate the given design as much as you can. :smile: :wave: 

- Your current design Screenshot:
Selection_042

- Figma
Screenshot from 2023-05-17 01-19-46
 design:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @lily4178993 , 

# Almost there 🟡 

1094a3e620ebea4cd6cf2d0aceaa8f52


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing some changes from the previous reviewers, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
-  Readme looks very professional✔
- Excellent communication with the reviewer💯 🥇 ✔

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily4178993** [comment]
Thank you @Nemwel-Boniface for your review.
I will implement the new event you suggested.

**Review by roshan-bajgain — APPROVED** [review]
Hi @lily4178993, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Your design looks similar to Figma design ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily4178993** [comment]
Thank you @roshan-bajgain for your review.

**Pull request merged**